Makefile

```diff
@@ -494,6 +494,12 @@ ifneq ($(wildcard $(PIO_LIB)/libgptl\.*), )
 	LIBS += -lgptl
 endif
 
+ifneq "$(LAPACK)" ""
+        LIBS += -L$(LAPACK)/lib
+        LIBS += -llapack
+        LIBS += -lblas
```

Comment:
@hyungyukang I had to add this like to connect to the blas library as well. It appears that you did not need this flag when you compiled?

Reply:
@mark-petersen, without `LIBS += -lblas`, the model was compiled for some platforms (NERSC, MacOS, Ubuntu) for me. Did you fail to compile before you add it? If so, we should add it. Thanks!